### ISSUE TITLE
Illumination aligned with camera?

### ISSUE THREAD

**Issue by Kai-46** [YueJiang-nj/CVPR2020-SDFDiff#8]
Thanks for sharing the source code. I noticed [this line](https://github.com/YueJiang-nj/CVPR2020-SDFDiff/blob/d44883497cc8dd8bdf106fed408ea0a50b27af76/multi_view_code/code/main.py#L546) in your code. I just want to double check that in your multi-view code, the directional light for each training image is aligned with the camera, hence varies across different training views, right? In other words, the lighting is not fixed for the 26 training images? 

Thank you!

**Comment by YueJiang-nj** [comment]
Yes! The lighting is always aligned with the camera and not fixed
